.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
In the future, I would add these types of files in their own PR to keep everything clean. Especially since, if there was any discussion or justification required, this would have its own dedicated thread rather than adding a discussion - unrelated to #12 - in this PR dedicated to addressing #12. It's fine for this PR but something to keep in mind for the future.

Reply:
Yeah, I will remove this in this PR and make a new issue.